Samples/Windowing/cs-winui/Properties/launchSettings.json

```diff
@@ -0,0 +1,7 @@
+﻿{
+    "profiles": {
+        "Windowing": {
+            "commandName": "MsixPackage"
+        }
+    }
+}
```

Comment:
```suggestion
{
  "profiles": {
    "$projectname$ (Package)": {
      "commandName": "MsixPackage"
    },
    "$projectname$ (Unpackaged)": {
      "commandName": "Project"
    }
  }
}
```

Reply:
Might want to add the unpackaged configuration, like the WinAppSDK VSIX does: https://github.com/microsoft/WindowsAppSDK/blob/main/dev/VSIX/ProjectTemplates/Desktop/CSharp/SingleProjectPackagedApp/Properties/launchSettings.json